packages/esm-service-queues-app/src/transition-latest-queue-entry/add-patient-to-queue-modal/add-patient-to-queue-entry.modal.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Form, ModalBody, ModalFooter, ModalHeader, Stack, InlineNotification } from '@carbon/react';
+import { usePatient, useVisit, type Visit } from '@openmrs/esm-framework';
+import { useMutateQueueEntries } from '../../hooks/useQueueEntries';
+import { type QueueEntry } from '../../types';
+import QueueFields from '../../create-queue-entry/queue-fields/queue-fields.component';
+import styles from './add-patient-to-queue-entry.scss';
+
+interface AddPatientToQueueModalProps {
+  modalTitle: string;
+  activeVisit: Visit;
+  closeModal: () => void;
+}
+
+const AddPatientToQueueModal: React.FC<AddPatientToQueueModalProps> = ({ modalTitle, activeVisit, closeModal }) => {
+  const { t } = useTranslation();
+  const { mutateQueueEntries } = useMutateQueueEntries();
+
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [callback, setCallback] = useState<{
+    submitQueueEntry: (visit: Visit) => Promise<unknown>;
+  } | null>(null);
+
+  const handleSubmit = useCallback(
+    (event) => {
+      event.preventDefault();
+      setIsSubmitting(true);
+
+      callback
+        ?.submitQueueEntry?.(activeVisit)
+        ?.then(() => {
+          closeModal();
+          mutateQueueEntries();
+        })
+        ?.finally(() => {
+          setIsSubmitting(false);
+        });
+    },
+    [callback, activeVisit, closeModal, mutateQueueEntries],
+  );
+
+  return (
+    <Form onSubmit={handleSubmit} className={styles.container}>
+      <ModalHeader className={styles.modalHeader} closeModal={closeModal} title={modalTitle} />
+      <ModalBody>
+        <div className={styles.queueEntryActionModalBody}>
```

Comment:
```suggestion
        <div>
```
This classname doesn't exist in the stylesheet.